s4sim/hardware/config.py

```diff
@@ -450,7 +450,7 @@ def get_example():
     bnd["low"] = 255.5
     bnd["high"] = 300.5
     bnd["bandpass"] = ""
-    bnd["NET"] = 1281.0
+    bnd["NET"] = 2077.0
```

Comment:
That is a pretty big difference!

Reply:
The old one apparently was very similar to the SAT, but scaled for the different bands, but the LAT has much smaller pixels for the high frequencies (so it makes sense that the noise is much higher). There are many more pixels though so we get more averaging.